components/LanguageSelector.tsx

```diff
@@ -33,7 +34,7 @@ export const LanguageSelector: React.FC<LanguageSelectorProps> = (props) => {
   };
 
   return (
-    <View>
+    <View {...testIDProps('language')}>
```

Comment:
can we add the testID for picker instead of view as we are handling the labels in picker.tsx